### PR TITLE
🧹 remove unused flag for report output

### PR DESCRIPTION
**Pull request by chris-rock**
The pager wasn't working as expecting and the new defaults with web reporting and compact view do not require us to have that anymore. Lets remove it.

### CODE REVIEW

File: cli/reporter/print_report.go [hunk 1]
```diff
@@ -76,17 +75,7 @@ func (r *reportRenderer) print() error {
 		}
 	}
 
-	if r.usePager && pager.Supported(r.pager) {
```
Comment (preslavgerchev) [inline cli/reporter/print_report.go:79]: can we also remove those reportRenderer props: `usePager` and `r.pager`? looks like we will not be needing those

Comment (chris-rock) [reply cli/reporter/print_report.go:79]: Yes, good call

Comment (chris-rock) [reply cli/reporter/print_report.go:79]: Code updated and ready for re-review

**Review by czunker — APPROVED** [review]
LGTM Thanks @chris-rock

**Pull request merged**